src/main/java/entity/Post.java

```diff
@@ -24,12 +26,17 @@ public class Post {
     @NotEmpty
     private List<@Valid Exercise> exerciseList;
```

Comment:
Do we keep Exercises as POJO or maybe change to ids as in User class for posts etc..

Reply:
Yeah we can use ids